src/declare.rs

```diff
@@ -47,8 +47,8 @@ impl<'gcc, 'tcx> CodegenCx<'gcc, 'tcx> {
         unsafe { std::mem::transmute(func) }
     }*/
 
-    pub fn declare_global(&self, name: &str, ty: Type<'gcc>, is_tls: bool, link_section: Option<Symbol>) -> LValue<'gcc> {
-        let global = self.context.new_global(None, GlobalKind::Exported, ty, name);
+    pub fn declare_global(&self, name: &str, ty: Type<'gcc>, is_import: bool, is_tls: bool, link_section: Option<Symbol>) -> LValue<'gcc> {
```

Comment:
I'd rather make this function take a `GlobalKind` directly. That looks cleaner to me than a `true/false` value that you don't know what it is at the call site.

Reply:
DOne